src/web/app/components/question-types/question-statistics/num-scale-question-statistics.component.html

```diff
@@ -11,23 +11,53 @@
       <table class="table table-bordered striped margin-0">
         <thead>
         <tr>
-          <td>Team</td>
-          <td>Recipient</td>
-          <td><span ngbTooltip="Average of the visible responses">Average</span></td>
-          <td><span ngbTooltip="Maximum of the visible responses">Max</span></td>
-          <td><span ngbTooltip="Minimum of the visible responses">Min</span></td>
-          <td><span ngbTooltip="Average of the visible responses excluding recipient's own response to himself/herself">Average excluding self response</span></td>
+          <th class="sortable-header" (click)="sortNumericalScaleStatsRowModel(SortBy.TEAM_NAME)">
+              Team
+              <span *ngIf="teamToRecipientToScoresSortBy !== SortBy.TEAM_NAME"><i class="fas fa-sort"></i></span>
+              <span *ngIf="teamToRecipientToScoresSortBy === SortBy.TEAM_NAME && teamToRecipientToScoresSortOrder === SortOrder.DESC"><i class="fas fa-sort-down"></i></span>
+              <span *ngIf="teamToRecipientToScoresSortBy === SortBy.TEAM_NAME && teamToRecipientToScoresSortOrder === SortOrder.ASC"><i class="fas fa-sort-up"></i></span>
```

Comment:
Do you think that we can have some sort of component like `SortableTable` that gets rid of the need for all these boilerplate code?

Reply:
Good idea. It worth to try and it is tricky to design it generically.

Reply:
Will try to do this for my MCQ table implementation.